joshuaCarpentier/assignments/slidingwindow/lc209/lc209.js

```diff
@@ -1,6 +1,9 @@
+
+
+
 function smallest_subarray_with_given_sum(s, arr) {
```

Comment:
Make sure you are writing out an explanation / high level summary of your approach and/or include detailed comments in your code.